flow/templates/torque.sh

```diff
@@ -1,15 +1,13 @@
 {% extends "base_script.sh" %}
 {% block header %}
 #PBS -N {{ id }}
+#PBS -l pmem={{ operations | _calc_memory(memory) }}G
```

Comment:
Submissions that do not define a memory directive should not add this line, so this needs to be set in a conditional block. See the section below with `{% if walltime %}` for an example.

Reply:
The references I found below use `mem` instead of `pmem` and expect the suffix to be `gb` or `GB`. I'm not sure what is best to use here.

https://www.osc.edu/supercomputing/batch-processing-at-osc/pbs-directives-summary
https://docs.pace.gatech.edu/faqs/pbs/

Reply:
For the command line flag, we ask users to enter memory to request per node. 
`pmem` -> Memory to request per process
`mem` -> Memory to request overall.

I am not sure what's the context of "per node" in our case. (I think it means total memory so we should use `mem`)

Reply:
I agree with your assessment. Let's use `mem`, we intend to request a total amount of memory (not an amount per core requested).